packages/frontend/amp/components/MainMedia.tsx

```diff
@@ -91,10 +91,10 @@ const mainImage = (element: ImageBlockElement): JSX.Element | null => {
                     <input
                         aria-checked={false}
                         type="checkbox"
-                        id="show-caption"
+                        aria-label="show-caption"
```

Comment:
Question on this - you removed the `id` and `for` attribute - these are standard semantics but does AMP do away with them?

Reply:
Interesting you should ask - according to the lighthouse reference docs, there are 4 ways to label inputs. 

> Methods for adding labels
> Create a label element wrapped around the input field. This creates an implicit, rather than explicit, association between the label and the input field.
> Associate the label element with the input field using the for attribute whose value is the ID attribute of the input. This creates an explicit, rather than implicit, association between the label and the input field.
> Use an aria-label attribute value.
> Use aria-labelledby attribute value.

Original code used option 2, and I could see these in the rendered HTML, but did this not seem to be passing the Lighthouse test. 
I tried all three other options and the standalone aria-label was the only one that passed the LightHouse audit. I am not sure why unfortunately

Reply:
Ah, so it's because there is no text contents in the label, just an SVG. You'd need to add a span and apply the screenReaderOnly style to it as per https://github.com/guardian/dotcom-rendering/blob/master/packages/frontend/web/components/Dateline.tsx#L21

The reason `for` and `id` approach works slightly better is because it means clicking the label will check the box, which possibly is required for this functionality to work!

Reply:
Tangential, but how does this sound on a screenreader? Is the hyphen announced (or am I misunderstanding what's happening!)?

Reply:
Nope, this should be a descriptive name :)